### PR TITLE
Components

### PR DESCRIPTION
**Pull request by damiecode**
Thank you for reviewing our work.
- [x] Prepare the directory structure 
- [x] Implement the App component 
- [x] Implement the BooksList component
- [x] Implement the Book component
- [x] Implement the BooksForm component
- [x] Define the Redux actions and reducers
- [x] Initialize the Redux store

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <BooksList />
+    <BooksForm />
+  </div>
+);
+
+export default App;
```
Comment (shloch) [inline src/App.js:13]: Please relocate this this file to `"src/components/"` folder. Has nothing to do with performance or workings of  the project, but this is how the guideline wants it structured. Hope you understand, thanks

File: src/components/BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BooksForm.js:2]: Relocate this component inside `"src/containers/"` folder ....... all containers, will be in this folder, thanks

File: src/components/BooksList.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from './Book';
+
```
Comment (shloch) [inline src/components/BooksList.js:5]: Relocate this file into `"src/containers"` folder

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @damiecode , @raheebwa 

All your Logic stands out properly....but please for conformity with guidelines of this project, you have 03 files to relocate to specific folders before you get a thumbs up

waiting for a last re-review

### Happy coding

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**